### ISSUE TITLE
reboot system regulary

### ISSUE THREAD

**Issue by adcurtin** [debloper/piosk#48]
I used to use chilipie kiosk, that rebooted the pi every day at 3am to prevent memory leaks and other issues from eventually causing problems. An option to automatically reboot the pi regularly would be helpful

**Comment by andiohn** [comment]
Cron job. Just make a cron job to reboot until we have the ability to schedule tasks from the interface. 

https://github.com/debloper/piosk/issues/28

**Comment by debloper** [comment]
Agreed... until the root cause of the 8hr crash is detected, it's better to just reboot it.

**Comment by tismofied** [comment]
> Agreed... until the root cause of the 8hr crash is detected, it's better to just reboot it.

Oohhh That's why I am finding the pi at the desktop and the kiosk mode is stopped 😳.
Is it because of browser memory leaks?
I use the kiosk to pull up grafana local dashboards.

**Comment by tismofied** [comment]
> Cron job. Just make a cron job to reboot until we have the ability to schedule tasks from the interface. 
> 
> 
> 
> https://github.com/debloper/piosk/issues/28
> 
> 
> 
> 

Would a cronjob like this be safe to use on my pi to reboots every 8 hours?
`0 */8 * * * /sbin/shutdown -r now`

**Comment by creative0100** [comment]
well that explains why I come into a desktop rather than piosk running!
